rfcs/2020-12-08-5448-dynamic-sampling.md

```diff
@@ -0,0 +1,351 @@
+# RFC 5448 - <2020-12-08> - Dynamic sampling in the `sampler` transform
```

Comment:
Just noting this RFC issue number is pointing to #5448 which doesn't seem right. I think you probably need to open a separate RFC issue like #2414 and then link that here